app/services/complete_moab_handler.rb

```diff
@@ -223,8 +227,7 @@ def update_online_version(status = nil, set_status_to_unexp_version = false, che
     transaction_ok = with_active_record_transaction_and_rescue do
       raise_rollback_if_cm_po_version_mismatch
 
-      # FIXME: what if there is more than one associated complete_moab?
-      if incoming_version > complete_moab.version && complete_moab.matches_po_current_version?
```

Comment:
per the commit message for this change, second half of the condition seemed unnecessary in light of the `raise_rollback_if_cm_po_version_mismatch` call above.